.gitignore

```diff
@@ -7,6 +7,9 @@ yarn-error.log*
 pnpm-debug.log*
 lerna-debug.log*
 
+
```

Comment:
Please remove package-lock.json from .gitignore! 
Now that we have some packages installed, we can run npm ci after pulling the development branch.